README.md

```diff
@@ -1,140 +1,127 @@
-### Creating a Python virtual environment
+For more information about AQA, see [The Aqua Book][aqua-book], and its [resources][aqua-book-resources].
 
-Creating a Python virtual environment depends on whether you are using [base Python](#base-python-interpreter) or
-[Anaconda](#anaconda-interpreter) as your interpreter.
+Rather than using different files and locations to store AQA and documentation, our intention is to centralise as much
+of this as possible onto your Git repository hosting platform, e.g. GitHub or GitLab.
 
-#### Base Python interpreter
+We use nudges, such as checklists in pull/merge requests, to minimise the burden on contributors and reviewers to
+complete AQA checks. This results in faster iterative development and deployment, whilst ensuring HM Government-wide
+standards on assurance are met.
 
-If you are using base Python, there are multiple ways to create virtual environments in Python using `pip`, including
-(but not limited to):
+We have also included [pre-commit hooks][pre-commit] to prevent accidental committing of secrets, large data files, and
+Jupyter notebook outputs for security purposes.
 
-- [`venv`](https://docs.python.org/3/tutorial/venv.html);
-- [`virtualenv`](https://virtualenv.pypa.io/en/stable/);
-- [`pipenv`](https://github.com/pypa/pipenv); and
-- [`pyenv`](https://github.com/pyenv/pyenv) with its `virtualenv` [plugin](https://github.com/pyenv/pyenv-virtualenv).
+## Getting started
 
-Follow the documentation of your chosen method to create a Python virtual environment.
+> ⚠️ Only Unix-based systems (macOS, Linux, ...), and Python projects for GitHub or GitLab are supported — feel free to
+> [contribute](#contributing) to support other operating systems/programming languages!
 
-#### Anaconda interpreter
+To use this template to start your next coding project, make sure your system meets the [requirements](#requirements).
 
-If you are using [Anaconda or `conda`](https://www.anaconda.com/), following their
-[documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) to set up a
-conda environment.
+Once you're all set up, open your terminal, navigate to the directory where you want your new repository to exist, and
+run the following commands:
 
-## Creating a new project
-
-To create a new project using this template, in the folder where you want your project to be located, run the following
-code in your terminal:
-
-```shell script
+```shell
 cookiecutter https://github.com/ukgovdatascience/govcookiecutter
 ```
 
-Then following the prompts in your terminal to create the project structure.
+Follow the prompts, and that's it — you've created your project structure! Post-creation, there are a few
+[mandatory changes](#changes-to-make-post-creation) to make, as well as a some
+[optional changes](#changes-to-consider-post-creation) you should also consider making.
+
+Otherwise, that's it — happy coding! 🎉
 
-## Installing pre-commit hooks
+### Requirements
```

Comment:
```suggestion
### Requirements to use the cookiecutter template
```

I'd be even more explicit so that it is clearer what is needed if you just want to use the template vs. you want to contribute to the ukgovdatascience cookiecutter project

Reply:
Resolved in commit [`79b15e3`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/79b15e386e5c4561c6d8393f52edf25568d1c366).